torch/_logging/_internal.py

```diff
@@ -116,28 +116,118 @@ def clear(self):
+
+    An artifact is a particular type of log message within a component.
+    A component can emit many different kinds of artifacts.
+
+    Keyword args:
+        dynamo (:class:`int`):
+            The log level for the TorchDynamo component. Default: ``logging.WARN``
+
+        aot (:class:`int`):
+            The log level for the AOTAutograd component. Default: ``logging.WARN``
+
+        inductor (:class:`int`):
+            The log level for the TorchInductor component. Default: ``logging.WARN``
+
+        bytecode (:class:`bool`):
+            Whether to emit the original and generated bytecode from TorchDynamo.
+            Default: ``False``
+
+        aot_graphs (:class:`bool`):
+            Whether to emit the graphs generated by AOTAutograd. Default: ``False``
+
+        aot_joint_graph (:class:`bool`):
+            Whether to emit "aot_joint_graph" log artifacts. Default: ``False``
+
+        graph (:class:`bool`):
+            Whether to emit the graph captured by TorchDynamo in tabular format.
+            Default: ``False``
+
+        graph_code (:class:`bool`):
+            Whether to emit the python source of the graph captured by TorchDynamo.
+            Default: ``False``
+
+        guards (:class:`bool`):
+            Whether to emit the guards generated by TorchDynamo for each compiled
+            function. Default: ``False``
+
+        recompiles (:class:`bool`):
+            Whether to emit a guard failure reason and message every time
+            TorchDynamo recompiles a function. Default: ``False``
+
+        output_code (:class:`bool`):
+            Whether to emit the TorchInductor output code. Default: ``False``
+
+        schedule (:class:`bool`):
+            Whether to emit the TorchInductor schedule. Default: ``False``
+
+        modules (dict):
+            This argument provides an alternate way to specify the above log
+            component and artifact settings, in the format of a keyword args
+            dictionary given as a single argument. There are two cases
+            where this is useful (1) if a new log component or artifact has
+            been registered but a keyword argument for it has not been added
+            to this function and (2) if the log level for an unregistered module
+            needs to be set. This can be done by providing the fully-qualified module
+            name as the key, with the log level as the value. Default: ``None``
+
+
+    Example::
+
+        >>> # xdoctest: +SKIP
+        >>> import logging
+
+        # The following changes the "dynamo" component to emit DEBUG-level
+        # logs, and to emit "graph_code" artifacts.
+
+        >>> torch._logging.set_logs(dynamo=logging.DEBUG, graph_code=True)
+
+        # The following accomplishes the same thing as the above example, but
+        # uses the `module` keyword arg.
+
+        >>> torch._logging.set_logs(modules={"dynamo":logging.DEBUG, "graph_code":True, "unregistered.module.name":logging.DEBUG})
```

Comment:
Let's be consistent with the above, and let's not encourage using `modules` for keywords.
```suggestion
        # The following enables the logs for a different module

        >>> torch._logging.set_logs(modules={"unregistered.module.name": logging.DEBUG})
```